lib/activerecord/all/activerecord.rbi

```diff
@@ -9,3 +9,9 @@ class ActiveRecord::Migration::Current < ActiveRecord::Migration
   def change_column(table_name, column_name, type, options = nil); end
   def create_table(table_name, options = nil); end
 end
+
+module ActiveRecord::AttributeMethods::Dirty
+  extend T::Sig
+  sig { params(attr_name: Symbol, options: T.untyped).returns(T::Boolean) }
+  def saved_change_to_attribute?(attr_name, **options); end
```

Comment:
This matches the API generated by sorbet in gems/ folder

  def saved_change_to_attribute?(attr_name, **options); end